### PR TITLE
update .eslintrc

### PR DESCRIPTION
**Pull request by MarcoWorms**
This is the proposal that we are using in pagarme/vtex. It's airbnb without semi, adding space after function declarations and adding dangling comma where it's needed.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,6 +1,17 @@
 {
-  "extends": "airbnb",
+  "extends": ["airbnb-base"],
```
Comment (mccraveiro) [inline .eslintrc]: Why `airbnb-base`?

Comment (MarcoWorms) [inline .eslintrc]: idk, we use it like this in pagarme-js and in vtex :P

Comment (mccraveiro) [inline .eslintrc]: @MarcoWorms it seems that `airbnb-base` does not have react rules

**Comment by MarcoWorms** [comment]
I'm reopening this

**Pull request closed without merging**